app/controllers/education_controller.rb

```diff
@@ -1,7 +1,17 @@
 class EducationController < ThemesController
 
   def index
-    @content = "Education"
+    @title = helpers.education_title
+
```

Comment:
Do you mind removing the extra empty lines please? 😅 
We usually like to keep lines related to the same thing close together, since those are all about populating the page with text.